aspnetcore/mvc/models/validation.md

```diff
@@ -57,6 +57,18 @@ MVC supports any attribute that derives from `ValidationAttribute` for validatio
 
 There may be instances where you need more features than built-in attributes provide. For those times, you can create custom validation attributes by deriving from `ValidationAttribute` or changing your model to implement `IValidatableObject`.
 
+## Notes on the use of the Required attribute
+
+Non-nullable [value types](/dotnet/csharp/language-reference/keywords/value-types) (such as `decimal`, `int`, `float`, and `DateTime`) are inherently required and don't need the `Required` attribute. The app performs no validation checks server-side for non-nullable types that are marked `Required`.
+
+MVC model binding, which isn't concerned with validation and validation attributes, rejects a form field submission containing a missing value or whitespace for a non-nullable type. Model binding usually ignores missing data for non-nullable types, where the form field is absent from the incoming form data.
```

Comment:
The word "usually" doesn't instill confidence in the reader. Under what circumstances does model binding not ignore missing data?

Reply:
That came from @dougbu ...

> The MVC model binding system rejects a submission containing an empty string for a non-Nullable value type. But, it usually ignores completely-missing data for these properties.

Reply:
@dougbu I'm hoping to get clarification on why the word "usually" is used here.

Reply:
@scottaddie "usually" ➡️ "in the absence of a `[BindRequired]` attribute on the target property".

Reply:
@guardrex Please switch to this verbiage from @dougbu, and then this PR looks good.

Reply:
Reread the section and am wondering if placing the `[BindRequired]` paragraph right after this one would help? The client-side paragraph between two paragraphs about server-side validation may be making the "usually" meaning less clear -- it's not followed immediately by a "but if you want to force this&hellip;".